CameraJoint.gd

```diff
@@ -0,0 +1,31 @@
+extends SpringArm
+
+
+var mouse_sensitivity = 0.5
+var scroll_speed = 40
+
+
+# Called when the node enters the scene tree for the first time.
+func _ready():
+	set_as_toplevel(true)
+	spring_length = 7
```

Comment:
(Q based on my language ignorance:) Is there a reason some variables are initialized above at global scope and some here?

Reply:
The variables initialized above (`mouse_sensitivity`, `scroll_speed`) are variables that were defined by me to be used in this `CameraJoint.gd` file. `spring_length` is a property of the `CameraJoint` object that I'm setting when the scene is initialized (not defined by me, so I can't put it above)